packages/cloud-functions/src/cico/Provider.ts

```diff
@@ -4,3 +4,9 @@ export enum Provider {
   Ramp = 'Ramp',
   Transak = 'Transak',
 }
+
+export enum CashInStatus {
```

Comment:
Do you think it could make sense to store more events than just these 3? These are the most important but it might be helpful to understand what steps in the CICO process are taking the longest for different providers. We'll probably need some amount of standardization to make apples-for-apples comparisons

Reply:
Yes, that's a good idea. It will require some working with product to see what to send and where to put it. It might make sense to do no standarization and just have a separate table for each provider where we put all the information of each provider, so we have the main one with these three events and specific ones for each provider with more details.

I'm trying to wrap up the work in the CICO bubble with these two webhooks and the other ones that are missing. Is it OK if I leave the simple version for now and we make another issue to add more information for each provider?